.github/workflows/benchmark.yml

```diff
@@ -44,6 +44,7 @@ jobs:
         if: contains( github.event.pull_request.labels.*.name, 'needs-benchmarks')
         run: .maintain/check_runtime.sh
 
+      # TODO: use nix
```

Comment:
For a later PR? Do we have a tracking issue?

Reply:
not issue. I can track it so. here we should solve impurity to allow output of nix to be cp back onto src files (modify rs)